sourcegraph/sourcegraph
# GitHub App worker: Sync Apps

Closes #54066 - Deletes GitHub Apps when they don't exist

## Test plan
- Create a GitHub App, delete GitHub App on GitHub, refresh worker, ensure Sourcegraph updates deleted App
- Ensure no regressions in the GitHub App **Installations** worker functionality
